### PR TITLE
Regenerated Network SDK

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Description

---

https://github.com/Azure/azure-rest-api-specs/commit/5ceb9c24160362fca070a168708e24b8ce3dc5b0

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: @EvgenyAgafonchikov add package release notes as below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
There was email from samer about holding off on network SDK until the spec is merged. I see the spec has been merged now, waiting for CI to finish.

**Comment by shahabhijeet** [comment]
@EvgenyAgafonchikov pelase reopen this PR (I can help to reopen) or create a new PR once you address conflicts and fix build failures. Closing for now.

**Comment by EvgenyAgafonchikov** [comment]
@shahabhijeet looks like this PR https://github.com/Azure/azure-sdk-for-net/pull/3853 contains changes from the current. So let's keep it closed.

**Pull request closed without merging**
